packages/synapse-react-client/src/components/SynapseTable/datasets/DatasetItemsEditor.tsx

```diff
@@ -705,14 +788,18 @@ export function DatasetItemsEditor(props: DatasetItemsEditorProps) {
           Cancel
         </Button>
 
-        <Button
-          disabled={!datasetToUpdate}
-          variant="contained"
-          color="primary"
-          onClick={() => mutate(datasetToUpdate!)}
-        >
-          Save
-        </Button>
+        <Tooltip title={!hasChangedSinceLastSave && NO_CHANGES_MADE}>
+          <div>
+            <Button
+              disabled={!datasetToUpdate || !hasChangedSinceLastSave}
+              variant="contained"
+              color="primary"
+              onClick={() => mutate(datasetToUpdate!)}
+            >
+              Save
+            </Button>
+          </div>
+        </Tooltip>
```

Comment:
Update 'Save' button to only enable when changes are pending. Add tooltip + div wrapper to tell the user why it's disabled.